miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -178,6 +188,82 @@ export.set_account_item
     # => [R', V]
 end
 
+#! Returns VALUE located under specified KEY in map in specified account storage slot. 
+#! Panics if 
+#! - the index is out of bounds.
```

Comment:
I think we mean if the value is >255. 

- There is a u32 assert in the get_acct_storage_slot_type_data procedure.
- But the procedure will also fail if the index is 257, because of the failing mtree_get in account::get_item

Changed it.

Reply:
I added a >255 since this is our max storage slot